### DISCUSSION TITLE
Student Development Pack Issue

### DISCUSSION THREAD

**Discussion by CFokstuen** [community/community#48451] [GitHub Education] [answered]
Greetings, I am attempting to join the GitHub Global Campus and gain access to the Student Developer Pack, however I cannot proceed past the "Upload proof of your academic status" part.

I am a current student in data science at an Australian university, and have evidence to support this.  I am experiencing difficulties on the page titled: Upload proof of your academic status, and I cannot upload evidence of enrollment.  Where it asks to "take a picture" it does not work and there does not appear to be any way to upload evidence.  I have enabled camera permissions for the site on my internet browser (Firefox), and I still cannot progress.  Further, I have attempted this through my phone, and am still without success.  Is there something I am missing?

![image](https://user-images.githubusercontent.com/118830275/221080153-2557da6a-c4a9-49e5-ab5b-2979c92d3e2e.png)

**Comment by samus-aran** [comment]
Hi @CFokstuen 👋 

Are you still experiencing this? How did you get on?

**Comment by Luevateros** [comment]
I'm also experiencing this issue. I tried with my phone. The page gives me two options: camera 2 0 (facing back) or camera 2 1 (facing front). No matter what I choose, both options select the front camera. My phone's front camera takes low-resolution photos, and all photos of my ID are blurred. If I could choose the back camera, I would be able to take a clear photo of my ID. I have a Huawei nova 3. Any help is appreciated.

**Comment by CFokstuen** [comment, marked as answer]
Thanks @samus-aran for you inquiry, I got on well.

I have managed to find a solution to my posted problem.  I was at first using my default browser (Firefox) with all permissions allowed for Github.  As it turns out this is not the browser to use here.  I found success by switching to MS Edge, and was able to progress with my application without any hiccups.

Also, @Luevateros I hope you have made progress, and if not perhaps switching web browsers may help to progress.  Just be mindful that when I applied I used two forms of enrollment proof as i have read that there are some applicants that have not met the evidence requirement with one piece of proof.  Good luck.

**Reply by samus-aran** [reply]
Thank you for returning and providing the solution to your problem @CFokstuen . Certainly this could help others in similar situations. Enjoy your Global Campus Benefits 👍 Anything in particular you were looking to try out first?

**Reply by CFokstuen** [reply]
Thanks, @samus-aran, I do hope that it can assist others into the future.  I am just dipping my toes into Pycharm and VS Code.  Copilot is a cool feature, but in the next several months when i have some spare time I'm looking forward to exploring the Web Dev kit more.
